core/storage/statistics/gae_models.py

```diff
@@ -2161,95 +2129,6 @@ def get_deletion_policy():
         """Model doesn't contain any data directly corresponding to a user."""
         return base_models.DELETION_POLICY.NOT_APPLICABLE
 
-    @classmethod
-    def create_or_update(
```

Comment:
Why is this (and the following) deleted?

Reply:
Ditto